src/main/java/edu/harvard/iq/dataverse/search/IndexServiceBean.java

```diff
@@ -897,7 +897,8 @@ public SolrInputDocuments toSolrDocs(IndexableDataset indexableDataset, Set<Long
 
         if (dataset.isHarvested()) {
             solrInputDocument.addField(SearchFields.IS_HARVESTED, true);
-            solrInputDocument.addField(SearchFields.METADATA_SOURCE, HARVESTED);
+            solrInputDocument.addField(SearchFields.METADATA_SOURCE,
```

Comment:
Looks good. Probably need a release note about this feature that also notes that (async/background) reindexing is needed to populate the facet.

Reply:
Hi @qqmyers 👋🏼 

Thanks! I will add the release note in a few just looking at some other PR right now but I will add it ASAP. Regarding the tests, I am not sure I will also look at this. 

Best,
Juan

Reply:
Note added and test condition added to the harvesting test to search by the new collection name. 😃

Reply:
Putting what I said during standup in writing: 
Rather than using the HarvestingClient's nickname for this facet, we should probably use the name of the local collection into which the client is harvesting. 
The clear advantages of doing that: 
1) This mirrors what we are doing for the local datasets: ```solrInputDocument.addField(SearchFields.METADATA_SOURCE, rootDataverse.getName());```
2) The name of the collection is likely to be more descriptive/better-looking to a human user
3) While both the client nickname and the name of the local collection are chosen by the local admin, it is far easier to change the latter.  The former is not editable at all. With the current implementation, if a prod. instance admin realizes that they named the harvesting client ```oai_3``` and that's what's going to show up in the facet, the only way for them to address it would be to delete the client (and all the content associated with it), and re-create it with a better-looking nickname, then re-harvest. 
4) This will make it unnecessary to add an extra field with a descriptive label to the client class (as was mentioned during standup).